src/components/v5/shared/Button/Button.tsx

```diff
@@ -28,6 +28,7 @@ const Button: FC<PropsWithChildren<ButtonProps>> = ({
   iconName,
   iconSize = 'tiny',
   isIconRight,
+  isFilled,
```

Comment:
Please remove this prop and just use another mode

Reply:
Yeah, you can create another one :)

Reply:
added new prop and fixed